static/docs/user-guide/dvcignore.md

```diff
@@ -18,26 +19,126 @@ provide similar functionality as `.gitignore` files provide for `git`.
 - Each line should contain only one pattern;
 - During execution of commands that traverse directories, DVC will ignore
   matching paths;
```

Comment:
traverse directories - it's not only about dirs, right?

Reply:
Ignore is not only about dirs, but it works only during traversing of dirs :)